pepper-apis/config/DataDonationPlatform.yaml.ctmpl

```diff
@@ -3,10 +3,9 @@
 service: pepper-backend
 runtime: java11
 instance_class: B8
-{{if eq $environment "prod"}}
-basic_scaling:
-  max_instances: 6
-  idle_timeout: 45m
+{{if (or (eq $environment "prod") (eq $environment "staging"))}}
```

Comment:
How about
```
if prod:
  manual_scaling, 3
elif staging:
  manual_scaling, 1
else:
  basic_scaling
```
since we don't really use staging that often?

Reply:
d4be435ea35a5e12c32c7993cd5251a7456a1754